test/general/test_fill.py

```diff
@@ -46,8 +46,8 @@ class PlayerDefinition(object):
     basic_items: List[Item]
     regions: List[Region]
 
-    def __init__(self, world: MultiWorld, id: int, menu: Region, locations: List[Location] = [], prog_items: List[Item] = [], basic_items: List[Item] = []):
-        self.multiworld = world
+    def __init__(self, multi_world: MultiWorld, id: int, menu: Region, locations: List[Location] = [], prog_items: List[Item] = [], basic_items: List[Item] = []):
+        self.multiworld = multi_world
```

Comment:
As said on discord, please change those to `multiworld` as well.
(I would be in favor to replace all the other `multi_world`s by `multiworld`s in this file as well).